tests/unit/test_activation_checkpointing.py

```diff
@@ -0,0 +1,147 @@
+# TODO: add tests with model parallelism for activation partitioning and other features.
+
+from copy import deepcopy
+
+import pytest
+
+import torch
+
+import deepspeed
+ckpt = deepspeed.checkpointing.checkpoint
```

Comment:
Out of curiosity, is there a difference to 
`import deepseed.checkpointing.checkpoint as ckpt`

Reply:
I started with that (which is way nicer), but found that we can't actually do that today:

![image](https://user-images.githubusercontent.com/620322/93627479-de1d5c00-f999-11ea-833b-bfe46b2415a5.png)

It's due to how we create `deepspeed.checkpoint` by importing from runtime in `deepspeed/__init__.py` versus having a `deepspeed/checkpoint/__init__.py` that imports from runtime.

I think it would be nice to change this. There's an alternative implementation with `deepspeed.pipe`, but the tradeoff is we have an extra `deepspeed/pipe/` directory hanging around.

Reply:
Thanks for sharing this tip. I am glad I asked.